hivemind/hivemind_cli/run_server.py

```diff
@@ -56,6 +56,9 @@ def main():
     parser.add_argument('--stats_report_interval', type=int, required=False,
                         help='Interval between two reports of batch processing performance statistics')
 
+    parser.add_argument('--custom_module_path', type=str, default=None, required=False,
```

Comment:
Specifying `default` isn't necessary when required=False

Reply:
Fixed. Yet there are four more parser parameters which have default=None and required=False specified.